grails-app/conf/Config.groovy

```diff
@@ -291,14 +291,10 @@ marvl {
 // WFS indexed file info
 indexedFile.fileSizeColumnName = "size"
 
-// Something else is adding an appender (what?!) - get rid of it, else we end up
-// with duplicate logs in production.
-org.apache.log4j.Logger.rootLogger.removeAllAppenders()
```

Comment:
Alright, then forget about my comment then :)

Because at the time I realy wanted to remove all that, but I noticed that it actually resulted in double log lines.